lib/nine_one_one/pager_duty_service.rb

```diff
@@ -26,6 +28,10 @@ def trigger_event(incident_key, description, details_hash = nil)
       # rubocop:enable Style/GuardClause
     end
 
+    def trigger_low_urgency_event(description, source, details_hash = {})
```

Comment:
This is a minor thing - how do you feel about having three methods:
 - generic `trigger_event`, where you specify severity (not optional),
 - `trigger_low_urgency_event`
 - `trigger_high_urgency_event` (for completeness sake)
NineOneOne module would use trigger_high_urgency (at least for now).

I find the lack of symmetry disturbing ;).

Reply:
I lost in counting versions of the implementation but it in one of them I had it ;) I simply added an alias to the trigger_event method. If I revert it, would it be okay with you?:>